src/Apps/Artist/Routes/RelatedArtists/index.tsx

```diff
@@ -15,33 +14,12 @@ export class RelatedArtistsRoute extends Component<RelatedArtistsProps> {
 
     return (
       <React.Fragment>
-        <Sans size="3" weight="medium">
-          Related
-        </Sans>
-
-        <Spacer mb={2} />
-
         <RelatedArtistsList
           kind={"MAIN"}
           artist={viewer.mainArtists as any}
           scrollTo="#jumpto-RouteTabs"
         />
 
-        <Spacer mb={3} />
-        <span id="jumpto-RelatedArtists-Contemporary" />
-
-        <Sans size="3" weight="medium">
-          Suggested contemporary
-        </Sans>
-
-        <Spacer mb={3} />
-
-        <RelatedArtistsList
-          kind={"CONTEMPORARY"}
-          artist={viewer.mainArtists as any}
-          scrollTo="#jumpto-RelatedArtists-Contemporary"
-        />
```

Comment:
@mzikherman So, the design didn't have anything like this in it. I'm not sure exactly what the behavior should be, but this can always be restored if we find it's needed. As is there's still some cleanup that needs to happen here.

Reply:
Yea, I think this needs to be put back. We're still fetching all the contemporary artists but not displaying it. So if you're going to wipe out this section wipe out the data fetching as well.

Also, I discussed with @briansw in slack but I think there _should_ be two sections here: Main and Contemporary. The comps simply didn't have that correctly (I think).